tests/test_cmdline.py

```diff
@@ -227,7 +227,7 @@ def test_dev_command(
 
     assert isinstance(cmd, DevCommand)
     assert cmd.platform == "macOS"
-    assert cmd.output_format is None
+    assert cmd.output_format == "app"
```

Comment:
This change is definitely required; but we should also check some other command line manifestations - `briefcase dev`, `briefcase dev macOS`, and `briefcase dev macOS Xcode` all need to work.